src/click/core.py

```diff
@@ -2343,17 +2345,17 @@ def handle_parse_result(
                 value = None
 
         if self.expose_value:
-            ctx.params[self.name] = value  # type: ignore
+            ctx.params[self.name] = value
 
         return value, args
 
     def get_help_record(self, ctx: Context) -> tuple[str, str] | None:
-        pass
```

Comment:
Linter complains about not having an implementation when it's an abstract base class. Using `return None` makes it not do that.